### ISSUE TITLE
Better Windows Support

### ISSUE THREAD

**Issue by jyaworski** [voxpupuli/modulesync_config#139]
Looking at puppet-iis, our Windows support is awful. We should make it so they don't need to jump through so many hoops. Maybe we have a windows_supported/unix_supported/etc bool that controls which OS we test against. @petems @bastelfreak @rnelson0

**Comment by juniorsysadmin** [comment]
In .sync.yml we also need to exclude the non-Windows nodesets for Windows-only modules.

**Comment by jyaworski** [comment]
That was my main point with those Windows-only modules. Why test with anything else?

**Comment by TraGicCode** [comment]
@voxpupuli/collaborators  I do alot of windows development.  Do you guys have a problem see an issue with me the following boxes to our nodesets?
```
opentable/win-2008r2-standard-amd64-nocm
opentable/win-2012-standard-amd64-nocm
opentable/win-2012r2-standard-amd64-nocm
```

**Comment by ekohl** [comment]
@TraGicCode looks like people are generally happy with adding the nodesets.

**Comment by wyardley** [comment]
I merged #378
